spread.yaml

```
@@ -542,7 +568,7 @@ prepare: |
         fi
     fi
 
-    if [[ "$SPREAD_SYSTEM" == debian-* ]]; then
+    if [[ "$SPREAD_SYSTEM" == debian-* ]] || ( [[ "$SPREAD_SYSTEM" == ubuntu-* ]] && [ "$SPREAD_BACKEND" == "classic-desktop" ] ); then
```

Classic `[` aka `test` compares strings using single `=`


```suggestion
    if [[ "$SPREAD_SYSTEM" == debian-* ]] || ( [[ "$SPREAD_SYSTEM" == ubuntu-* ]] && [ "$SPREAD_BACKEND" = "classic-desktop" ] ); then
```

---

done